.eslintrc

```diff
@@ -143,6 +143,7 @@
       "unit-test-utils.ajs.ts",
       "mathjax.directive.ts",
       "math-expression-content-editor.component.ts",
+      "rte-output-display.component.spec.ts",
```

Comment:
Why is it added here?

Reply:
Instead of disabling catch in the entire file maybe use a bypass flag and state the reason for using the bypass flag through code-comment!

Reply:
Done!